packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -115,14 +116,26 @@ export class DeltaStorageService implements IDeltaStorageService {
 			async (event) => {
 				const restWrapper = await this.getRestWrapper();
 				const url = this.getDeltaStorageUrl();
-				const response = await restWrapper.get<ISequencedDocumentMessage[]>(url, {
-					from: from - 1,
-					to,
-				});
+				const response = await restWrapper.get<IR11sResponse<ISequencedDocumentMessage[]>>(
+					url,
+					{
+						from: from - 1,
+						to,
+					},
+					undefined,
```

Comment:
can you please add comments for argument names? I.e. 

undefined, // name
true,  // name

Reply:
Same for other places